recipe/meta.yaml

```diff
@@ -10,32 +10,43 @@ source:
 
 build:
   number: 0
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py<39]
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python
     - pip
     - flit-core >=3.7
   run:
-    - python >={{ python_min }}
+    - python
 
 test:
+  source_files:
+    - tests
   imports:
     - roman_numerals
   commands:
     - pip check
+    - ls
```

Comment:
Curious why this was added?

Reply:
Strange! Def a typo. I'll have get it removed 🫡